### PR TITLE
Kind property reference

### PR DESCRIPTION
**Pull request by btardif**
@ammanifold hey can you do a quick review for me.

### CODE REVIEW

File: Things_You_Should_Know/kind_property.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# App Service Kind property
+
+Everything in Azure is modeled as a **resource**. You can learn more about resources through [Azure Resource Manager documentation](https://docs.microsoft.com/azure/azure-resource-manager/management/overview)
+
+Every Azure resource has a set of generic properties that are common across all resources. **kind** is one of this generic azure resource properties. See [Resources - Get](https://docs.microsoft.com/rest/api/resources/resources/get)
```
Comment (ammanifold) [inline Things_You_Should_Know/kind_property.md:5]: Should **kind** be **Kind** ?

File: Things_You_Should_Know/kind_property.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+There are several different offerings in the azure portal that use the same underlying `Microsoft.web\sites` resource. [Learn more about `microsoft.web\sites`](https://docs.microsoft.com/azure/templates/microsoft.web/2019-08-01/sites)
```
Comment (ammanifold) [inline Things_You_Should_Know/kind_property.md:7]: In general, should this be `Microsoft.Web\Sites` ?

File: Things_You_Should_Know/kind_property.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+**App Service** uses the value of the **kind property** to specialize the UX of a `Microsoft.web\sites` resource in the Azure Portal.
+
+The `kind` property of an app is set during create. The portal does this based on the creat flow of your choice and or the configuration you enter. This is also true of the [App Service CLI](https://docs.microsoft.com/cli/azure/appservice?view=azure-cli-latest) and other clients like [Visual Studio Code](https://code.visualstudio.com/).
```
Comment (ammanifold) [inline Things_You_Should_Know/kind_property.md:11]: create flow

Comment (ammanifold) [reply Things_You_Should_Know/kind_property.md:11]: and/or

File: Things_You_Should_Know/kind_property.md (hunk 1 above)
Comment (ammanifold) [reply Things_You_Should_Know/kind_property.md:5]: here since it starts the sentence

Comment (btardif) [reply Things_You_Should_Know/kind_property.md:5]: Fixed

File: Things_You_Should_Know/kind_property.md (hunk 2 above)
Comment (btardif) [reply Things_You_Should_Know/kind_property.md:7]: Fixed

**Pull request merged**